autoload/goyo.vim

```diff
@@ -33,8 +33,12 @@ function! s:get_color(group, attr)
 endfunction
 
 function! s:set_color(group, attr, color)
-  let gui = a:color =~ '^#'
-  execute printf('hi %s %s%s=%s', a:group, gui ? 'gui' : 'cterm', a:attr, a:color)
+  if empty(a:attr)
```

Comment:
1. We also have to check `has('gui_running')`
1. `termguicolors` is a recent addition to Vim. We need to check if it's available first not to break Goyo on old versions of Vim.
1. I'd like to use the fullname `termguicolors` instead.

So it becomes `has('gui_running') || has('termguicolors') && &termguicolors`